config-dist.py

```diff
@@ -0,0 +1,13 @@
+# Do not change this directory path
+DIRECTORY = "sync/static/feeds/"
+
+DIGEST_AUTH = {
+    "username": "my_usename",
```

Comment:
Use admin / opencast here for username/pwd which would make it work with stable.opencast.org by default